Assets/BossRoom/Scripts/Game/ConnectionManagement/ClientGameNetPortal.cs

```diff
@@ -244,35 +243,18 @@ private IEnumerator TryToReconnect(string lobbyCode)
         /// <param name="port">The port of the host to connect to. </param>
         public void StartClient(string ipaddress, int port)
         {
-            var chosenTransport = NetworkManager.Singleton.gameObject.GetComponent<TransportPicker>().IpHostTransport;
-            NetworkManager.Singleton.NetworkConfig.NetworkTransport = chosenTransport;
+            //var chosenTransport = NetworkManager.Singleton.gameObject.GetComponent<TransportPicker>().IpHostTransport;
+            //NetworkManager.Singleton.NetworkConfig.NetworkTransport = chosenTransport;
 
-            switch (chosenTransport)
-            {
-                case UNetTransport unetTransport:
-                    unetTransport.ConnectAddress = ipaddress;
-                    unetTransport.ConnectPort = port;
-                    break;
-                case UnityTransport unityTransport:
-                    // TODO: once this is exposed in the adapter we will be able to change it
-                    unityTransport.SetConnectionData(ipaddress, (ushort)port);
-                    break;
-                default:
-                    throw new ArgumentOutOfRangeException(nameof(chosenTransport));
-            }
+            var utp = (UnityTransport) NetworkManager.Singleton.NetworkConfig.NetworkTransport;
 
-            ConnectClient();
-        }
+            // TODO: once this is exposed in the adapter we will be able to change it
```

Comment:
is that todo still relevant?

Reply:
I suppose it meant to eventually remove the need to cast to UnityTransport if this method was exposed in NetworkTransport? If so, then I don't think it is still relevant

Reply:
oh you're right, I remember this conversation. No need to clean it up just yet, I'll poke a few folks, see if we can standardize this.

Reply:
from this conversation, it's still a thing we want to address
https://unity.slack.com/archives/C01FWMSBZR7/p1651785925521629
Let's keep that todo